src/main/java/com/facebook/openwifirrm/ucentral/prov/models/RrmAlgorithmDetails.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifirrm.ucentral.prov.models;
+
+public class RrmAlgorithmDetails {
```

Comment:
You can capitalize this name as `RRMAlgorithmDetails`, see [src/RESTObjects/RESTAPI_ProvObjects.h](https://github.com/Telecominfraproject/wlan-cloud-owprov/pull/55/files#diff-69a248eae759933adffac236c046f0c279e3b65852b5320d2ef043966d0234f3)